testspiders/spiders/broken_link.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf8
```

Comment:
I would prefer inline docuemntation explaining the spider. ( can bring the readme content here ) and also inline documentation on what spider is trying to do at different decision points.